CHANGELOG.md

```diff
@@ -3,6 +3,102 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.6
+
+_Dec 22, 2023_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Improve Russian (ru-RU) locale on the Data Grid
+- 🐞 Bugfixes
+
+From https://github.com/mui/mui-x/pull/11330
+
+## Breaking changes
+
+- The filter panel no longer uses the native version of the [`Select`](https://mui.com/material-ui/react-select/) component for all components.
+- The `getOptionValue` and `getOptionLabel` props were removed from the following components:
+
+  - `GridEditSingleSelectCell`
+  - `GridFilterInputSingleSelect`
+  - `GridFilterInputMultipleSingleSelect`
+
+  Use the `getOptionValue` and `getOptionLabel` properties on the `singleSelect` column definition instead:
+
+  ```tsx
+  const column: GridColDef = {
+    type: 'singleSelect',
+    field: 'country',
+    valueOptions: [
+      { code: 'BR', name: 'Brazil' },
+      { code: 'FR', name: 'France' },
+    ],
+    getOptionValue: (value: any) => value.code,
+    getOptionLabel: (value: any) => value.name,
+  };
+  ```
+
+From https://github.com/mui/mui-x/pull/7069
+
+### Breaking changes
```

Comment:
Same as what @cherniavskii wrote about the Data Grid breaking changes - these should be under the Pickers section

Reply:
What should be under the pickers section? IIUC, all of these are for the datagrid.

Reply:
Yes, there are no breaking changes for the pickers 👍🏻